cosmoz-autocomplete.html

```diff
@@ -1,34 +1,67 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../iron-flex-layout/iron-flex-layout-classes.html">
-<link rel="import" href="../iron-icons/iron-icons.html" />
-<link rel="import" href="../iron-icon/iron-icon.html" />
-<link rel="import" href="../iron-selector/iron-selector.html" />
-<link rel="import" href="../paper-input/paper-input.html" />
-<link rel="import" href="../paper-material/paper-material.html" />
-<link rel="import" href="../paper-spinner/paper-spinner.html" />
+<link rel="import" href="../iron-icons/iron-icons.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../iron-selector/iron-selector.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-material/paper-material.html">
+<link rel="import" href="../paper-spinner/paper-spinner.html">
 <link rel="import" href="../cosmoz-behaviors/cosmoz-templatehelper-behavior.html">
 <link rel="import" href="../cosmoz-i18next/cosmoz-i18next.html">
 <link rel="import" href="../iron-validatable-behavior/iron-validatable-behavior.html">
 <link rel="import" href="../iron-form-element-behavior/iron-form-element-behavior.html">
 
-
-
-
 <!--
-`<cosmoz-autocomplete>` is an auto-complete/typeahead multi-select
-
-### Usage
-
-*html example here*
+`<cosmoz-autocomplete>` is an autocomplete/typeahead element.
+
+It has such features as multi selection, autocompletion, validation,
+case-intensive capabilities and some other cool things.
+
+### Examples
```

Comment:
These kind of examples should either be made as inline-demos according to https://www.webcomponents.org/publish (example at https://raw.githubusercontent.com/PolymerElements/paper-button/master/README.md) or moved to the demos. (`demo/`)